File: Programmation/TD3/ex4.pl
#!/usr/bin/perl

# How many times do we have $value ?

use strict ;
use locale ;

my @tableau=(1,2,3,5,0,4) ;
my $tableau ;
my $value = 0 ;
my $total = 0 ;

foreach $tableau(@tableau)
{
    if ($value == $tableau)
    {
	$total = $total + 1 ;
    }
}

print "On trouve " . $total . " fois \"" . $value . "\" dans \@tableau\n" ;

File: Programmation/TD3/ex4.rb
#!/usr/bin/ruby

array=[1,2,0,3,3,5,0,4]
value_to_search = 3
total = 0

array.each { |value| total += 1 if value == value_to_search}

# or :
# array.each do |value|
#       total += 1 if value == value_to_search
# end

puts "\"#{value_to_search}\" is #{total} time(s) in array"
